addons/mail_bot/views/res_users_views.xml

```diff
@@ -28,16 +28,17 @@
         </field>
     </record>
 
-    <!-- Blocked notification alert -->
-    <record id="view_users_form_inherit_mail_notification_alert" model="ir.ui.view">
-        <field name="name">res.users.preferences.form.notification.alert</field>
+    <!-- Update Preferences form !-->
+    <record id="res_users_view_form_preferences" model="ir.ui.view">
+        <field name="name">res.users.view.form.preferences.mail_bot</field>
         <field name="model">res.users</field>
-        <field name="inherit_id" ref="base.view_users_form_simple_modif"/>
+        <field name="inherit_id" ref="mail.view_users_form_simple_modif_mail"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='image_1920']" position="before">
-                <widget name="notification_alert"/>
-            </xpath>
+            <data>
+                <field name="notification_type" position="after">
+                    <field name="odoobot_state" readonly="0" groups="base.group_no_one"/>
+                </field>
+            </data>
```

Comment:
I don't understand why keep this diff. `view_users_form_inherit_mail_notification_alert` should be moved to `mail` and simply removed from here.

There is already `res_users_view_form` in `mail_bot` for the `odoobot_state` stuff.

Unless there is something I'm missing, there should be no "green" (+) diff on this file.